### PR TITLE
1.11 Storage Service Documentation 

### PR DESCRIPTION
**Pull request by anithamuthu79**
## Urgency
- [ ] Blocker 
- [ ] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

File: pages/services/storage-service/index.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+layout: layout.pug
+navigationTitle:  Release Notes for 1.10.5
```
Comment (gpaul) [inline pages/services/storage-service/index.md]: @gauripowale These say "release notes for 1.10.5". There is no storage service in 1.10.5.

Comment (gpaul) [reply pages/services/storage-service/index.md]: Ah, looks like this is a work in progress (https://github.com/mesosphere/dcos-docs-site/pull/614), please ignore me if you're still working on this section and feel free to message me for review or input of any kind.

Comment (gauripowale) [reply pages/services/storage-service/index.md]: @gpaul Yup, sorry @anithamuthu79 is currently working on adding the storage docs & will be changing this later. We've hidden the documentation from the site through this pull request: #614

Comment (gpaul) [reply pages/services/storage-service/index.md]: That's great, thanks @gauripowale and @anithamuthu79 for working on this. The source material is still somewhat in flux (e.g., the CLI help text changes as we flesh out the tests and functionality) so I will probably be opening related PRs once things settle down. I'd appreciate your reviews on those so don't hesitate to ask if you need something from me in the meantime.

**Pull request merged**